tests/test_nvfuser.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import logging
+
+import pytest
+import torch
+import torch.nn as nn
+from torch.cuda.amp.autocast_mode import autocast
+
+import xformers
+from xformers.components import Activation, LayerNormStyle
+from xformers.components.feedforward import build_feedforward
+
+_gpu_available = torch.cuda.is_available()
+
+xformers._is_functorch_available = True
+
+try:
```

Comment:
`pip install functorch` should probably be added [here](https://github.com/facebookresearch/xformers/blob/main/.circleci/config.yml)?